### PR TITLE
Adding typescript definitions and correcting some translations

### PR DESCRIPTION
**Pull request by joleb**

### CODE REVIEW

**Comment by thibaultdalban** [comment]
Thank you @joleb!
I will review it as soon as possible.

File: src/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+type Locale = "ar" | "de" | "en" | "es" | "fr" | "he" | "it" | "ja" | "ko" | "nl" | "pt-BR" | "ru" | "zh-Hans" | "zh-Hant"
```
Comment (thibaultdalban) [inline src/index.d.ts:1]: I'm not familiar with typescript definition, is there a way to use/import the values defined [here](https://github.com/Locale/amplify-i18n/blob/main/src/index.js#L4-L19)?
I'm pretty sure at a time the lists won't be up to date.

Comment (joleb) [reply src/index.d.ts:1]: I will take a look at this, I'm pretty certain it's possible :)

Comment (joleb) [reply src/index.d.ts:1]: Without using proper typescript it's sadly not possible using a .js-defined array as type. Neither I am able to import the array from a JSON-File and import this as a type. Thus, it's easier to convert the whole logic to typescript or maintain languages manually to the definition file.

**Comment by miguelpdiaz8** [comment]
Hi @thibaultdalban, will you merge this PR anytime soon? Thanks

**Review by thibaultdalban — APPROVED** [review]

**Comment by thibaultdalban** [comment]
@joleb thank you for your contribution <3

Released => `amplify-i18n@0.3.1`

**Pull request merged**